lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -50,43 +55,129 @@ define((require, exports, module) ->
       )
       return
 
-    callbackHidePopover = ->
-      popoverHandler.hide($("#formula"))
-      editor.off("changeSelection", callbackHidePopover)
-      editor.session.off("changeScrollTop", callbackHidePopover)
-      editor.session.off("changeScrollLeft", callbackHidePopover)
-      return
+    jqEditorContainer = $(editor.container)
+    jqFormula = -> $("#formula")
+
+    # ch stands for Context Handler
```

Comment:
This comments is now out of date, but you may want to add more descriptive one, with explanation what context handler is